src/entities/card/organisms/card-list.tsx

```diff
@@ -12,18 +12,52 @@ interface Props {
 }
 
 export const CardList: React.FC<Props> = ({ cards, getHref, loading }) => {
+  const containerRef = React.useRef<null | HTMLDivElement>(null);
```

Comment:
https://github.com/atomix-team/codestyle/blob/master/codestyle.md#11-access-to-internal-resources

Reply:
ffffffffffffffffffffffffffffffffffffffff